.travis.yml

```diff
@@ -1,76 +1,68 @@
-language: python
+language: c
 
-python:
-  - 2.7
+os:
+    - linux
+    - osx
 
-virtualenv:
-     system_site_packages: false
+cache:
+    directories:
+        $HOME/miniconda/pkgs
+        /Users/travis/miniconda3/pkgs
 
-# Configure the build environment. Global varibles are defined for all
-# configurations. Each configuration in the matrix is run for each python version
-# above.
+# Configure the build environment. Global varibles are defined for all configurations.
 env:
     global:
-        - PIP_WHEEL_COMMAND="--find-links http://sunpy.cadair.com/wheelhouse/index.html --use-wheel --upgrade --trusted-host sunpy.cadair.com"
+        - PYTHON_VERSION=2.7
         - TEST_MODE='offline'
         - LATEST_NUMPY=1.9.1
         - LATEST_PANDAS=0.15.2
         - NUMPY_VERSION=$LATEST_NUMPY
         - PANDAS_VERSION=$LATEST_PANDAS
 matrix:
-     include:
-        - python: 2.7
-          env: NUMPY_VERSION=1.8.1
-        - python: 2.7
-          env: PANDAS_VERSION=0.14.1
-        - python: 2.7
-          env: TEST_MODE='astropy-dev'
-        - python: 2.7
-          env: TEST_MODE='sphinx'
-        - python: 2.7
-          env: TEST_MODE='online'
+    include:
+#         - os: linux
+#           env: NUMPY_VERSION=1.8.1
+         - os: linux
+           env: PANDAS_VERSION=0.14.1
```

Comment:
LATEST_PANDAS is just the place where the latest version of pandas should be set. Same for numpy, it's just a naming thing to make it easier for people to modify.